website/docs/docs/guides/migration-guide/upgrading-to-0-19-0.md

```diff
@@ -42,19 +43,21 @@ See [dbt#2961](https://github.com/fishtown-analytics/dbt/pull/2961) for full imp
 ## New and changed documentation
 
 ### Core
-- [dbt-artifacts](dbt-artifacts): The JSON artifacts produced by dbt—manifest, catalog, run results, and sources—are simpler to consume and more clearly documented.
-- [snapshots](snapshots) ([invalidate_hard_deletes](invalidate_hard_deletes)): If the config `invalidate_hard_deletes` is enabled, `dbt snapshot` will update records whose unique key no longer exist in the snapshot query. Should those uniquely identified records "revive," `dbt snapshot` will re-add them.
+- [dbt Artifacts](dbt-artifacts): The JSON artifacts produced by dbt—manifest, catalog, run results, and sources—are simpler to consume and more clearly documented.
+- [dbt Classes](dbt-classes#result-objects), [on-run-end Context](on-run-end-context#results): The `Result` object has a new schema,
```

Comment:
Hanging comma, should this be a period instead?

Reply:
ty for catching!